packages/donor/src/components/basic/Accordion/AccordionPanel.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+  Accordion as AccordionMui,
+  AccordionSummary,
+  AccordionDetails,
+  Typography,
+} from "@mui/material";
+import styles from "./Accordion.module.scss";
+import { ReactComponent as MinusIcon } from "../../../assets/icons/minus.svg";
+import { ReactComponent as PlusIcon } from "../../../assets/icons/plus.svg";
+import AnchorTag from "../AnchorTag";
+import { Panel } from "./Accordion";
+
+interface IAccordionPanel {
+  expandedPanel: string | false;
+  panelData: Panel;
```

Comment:
What is this interface used for?
Why is this separate from AccordionPanelProps?